### ISSUE TITLE
Trying to upload project to d...

### ISSUE THREAD

**Issue by door43app** [unfoldingWord-dev/ts-desktop#669]
Notes
======
Trying to upload project to door43

Environment
======
Environment Key | Value
:--: | :--:
Version |9.0.0
Operating System | Windows_NT
Platform | win32
Release | 10.0.10586
Architecture | x64

Log History
======
```javascript
5/27/2016 4:26:25 PM E/ts-publish-publish.html:307:3: Error: Cannot push because a reference that you are trying to update on the remote contains commits that are not present locally.
6/16/2016 12:20:04 PM E/projects.js:380:3: Error: Command failed: C:\Program Files (x86)\translationStudio\resources\app\node_modules\trash\lib\win-trash.exe C:\Users\Chris\AppData\Local\translationstudio\targetTranslations\en-x-demo1_obs_text_obs

6/20/2016 11:21:34 AM E/ts-publish-publish.html:307:3: [object Object]

```

**Comment by cckozie** [comment]
Need to have a more descriptive error message.
![image](https://cloud.githubusercontent.com/assets/8171759/16202056/be512942-36d9-11e6-9015-d3d1ebc0d385.png)

**Comment by cdwhitfield73** [comment]
This is fixed.  A better error message will display, but also the problem creating this particular error was fixed.

**Comment by cckozie** [comment]
I don't know if this was the same error or not, but I'm not very fond of the error message!
![image](https://cloud.githubusercontent.com/assets/8171759/16271803/108dccbe-3861-11e6-821f-3a0040f57264.png)

![image](https://cloud.githubusercontent.com/assets/8171759/16271845/300cc7f2-3861-11e6-9eeb-979b0d93e43a.png)

**Comment by cdwhitfield73** [comment]
You are correct that it should fail when you upload this type of project.  We had written in code to check for this particular error and convert it to a user friendly message when we were using NodeGit.  Now that we changed to Git, the error message it returns is worded differently than what we were getting before, so it didn't catch it to convert it.  I have now fixed that issue so you will see the user friendly message.
